src/relay/backend/contrib/codegen_c/codegen_c.h

```diff
@@ -235,14 +289,14 @@ class CodegenCBase {
         continue;
       }
       this->PrintIndents();
-      code_stream_ << "std::memcpy(out" << i << ", " << outs[i].name << ", 4 * " << outs[i].size
+      code_stream_ << "memcpy(out" << i << ", " << outs[i].name << ", 4 * " << outs[i].size
```

Comment:
Did we use the std namespace somewhere?

Reply:
I dont think we need the namespace. Do we ?

Reply:
I think this explicitly avoids use of compiler builtins, which are sometimes perhaps broken?